src/main/resources/tanagra/cms_synpuf/expanded/entity/condition_occurrence.json

```diff
@@ -56,7 +56,7 @@
     "displayHint" : {
       "type" : "RANGE",
       "minVal" : 1.0,
-      "maxVal" : 2326856.0
+      "maxVal" : 20.0
```

Comment:
Same question about hardcoding. I would expect this to be generated with the indexer?

Reply:
Yeah, I should've mentioned up front that `expanded` can be ignored. Although it isn't bad to look at it a little to convince ourselves that the `EXPAND_CONFIG` step is working as expected.